### DISCUSSION TITLE
My Github Account is suspended

### DISCUSSION THREAD

**Discussion by Dev-Yoko** [community/community#110495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My Github account has been suspended for 2 years but there is no response from your side

Username: New-Dev3

**Comment by IanoNjuguna** [comment]
Sorry mate, what reason did they give for the suspension?

**Comment by gil-son** [comment]
Access your e-mail and search by: Github

Maybe you received a communication if its happened
